context/app/static/js/components/ProvTable.jsx

```diff
@@ -11,7 +11,7 @@ const FlexContainer = styled.div`
 `;
 
 function ProvTable(props) {
-  const { provData, typesToSplit } = props;
+  const { provData, assayMetaData, typesToSplit } = props;
```

Comment:
Prefer `Metadata`, with lowercase `d`. If it's an easy search-and-replace in this PR, great, otherwise file an issue.

Reply:
Sure, I was just using the naming conventions that were in place before I started working on this. I'll make the changes.